integration_tests/src/main/python/window_function_test.py

```diff
@@ -838,3 +837,16 @@ def do_it(spark):
             .withColumn('rn', f.count('c').over(window_spec))
 
     assert_gpu_fallback_collect(do_it, 'WindowExec')
+
+# In a distributed setup the order of the partitions returend might be different, so we must ignore the order
+# but small batch sizes can make sort very slow, so do the final order by locally
+@ignore_order(local=True)
+@pytest.mark.parametrize('ride_along', all_basic_gens + decimal_gens + array_gens_sample + struct_gens_sample + map_gens_sample, ids=idfn)
+def test_window_ride_along(ride_along):
+    assert_gpu_and_cpu_are_equal_sql(
+            lambda spark : gen_df(spark, [('a', LongRangeGen()), ('b', ride_along)]),
```

Comment:
Nit: indentation appears to be off here relative to other parameters being passed.